src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -92,7 +93,7 @@ public ScriptProjectSystem(OmniSharpWorkspace workspace, IOmniSharpEnvironment e
 
         public string Key => "Script";
         public string Language => LanguageNames.CSharp;
-        public IEnumerable<string> Extensions => new[] { ".csx" };
+        public IEnumerable<string> Extensions => new[] { CsxExtension };
```

Comment:
good point.
I changed from expression body property to getter only auto property